app/src/main/scala/org/alephium/explorer/api/model/Transaction.scala

```diff
@@ -32,10 +36,29 @@ final case class Transaction(
     outputs: ArraySeq[Output],
     gasAmount: Int,
     gasPrice: U256
-)
+) {
+  def toCsv(address: Address): String = {
+    val dateTime = Instant.ofEpochMilli(timestamp.millis)
```

Comment:
I think we should have UTC only and the conversion will be done by the front end. This endpoint is primarily meant to be called from a wallet, where it will  give you the option to export, last year, 3 monthes etc.

Reply:
We won't change the csv from the front-end side as far as I remember my discussions with @mvaivre, The file will be downloaded directly to the disk of the user

Reply:
For this first version I will rename the column to `dateTimeUTC` and open an issue about letting users choosing their timezone, we can wait for people requesting it.
Also [in excel you can easily make the conversion](https://www.extendoffice.com/documents/excel/3540-excel-convert-utc-gmt-to-local-time.html) if you want

Reply:
That's a very practical solution 👍 We could remove `${timestamp.millis}` from CSV as dateTimeUTC will be enough

Reply:
having both  `unixTimestamp` and `dateTime` comes form etherscan as mentioned  here https://github.com/alephium/explorer/issues/41 I don't have strong opinion on that, but I guess if they have both it's because users requested it?

Reply:
That's a strong enough evidence to convince me 😅